File: solutions/26. Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& a) {
        int j=0;
        if(a.size()==0 || a.size()==1)
            return a.size();
        for(int i=0;i<a.size()-1;i++){
            if(a[i]!=a[i+1])
                a[j++]=a[i];
        }
        a[j++]=a[a.size()-1];
        return j;
    }
};

File: solutions/26. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j=1
        for i in range(1,len(nums)):
            if nums[i]==nums[i-1]:
                continue
            else:
                nums[j]=nums[i]
                j+=1
        return j
